README.md

```diff
@@ -35,7 +41,7 @@ Alternatively, follow these steps to train it yourself.
 
```

Comment:
BTW, do you also wanna change the pretrained model URL for the one under an asset in the only release from the repo?